src/main/java/mate/academy/dao/impl/OrderDaoImpl.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.dao.impl;
+
+import java.util.List;
+import mate.academy.dao.OrderDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Order;
+import mate.academy.model.User;
+import mate.academy.util.HibernateUtil;
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
+    @Override
+    public Order add(Order order) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(order);
+            transaction.commit();
+            return order;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert an order: " + order, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("From Order o"
+                            + " left join fetch o.tickets t"
+                            + " left join fetch o.user u"
+                            + " left join fetch t.movieSession m"
+                            + " left join fetch m.movie"
+                            + " left join fetch m.cinemaHall"
+                            + " where o.user = :user", Order.class)
```

Comment:
```suggestion
            return session.createQuery("FROM Order o"
                            + " LEFT JOIN FETCH o.tickets t"
                            // fix other pls
                            + " left join fetch o.user u"
                            + " left join fetch t.movieSession m"
                            + " left join fetch m.movie"
                            + " left join fetch m.cinemaHall"
                            + " where o.user = :user", Order.class)
```

it is better to write SQL keywords in the upper case